File: applications/vehicle-telemetry-sink/src/main/java/showcase/IoT/connected/vehicles/streaming/sink/repository/VehicleTelemetryRepository.java
package showcase.IoT.connected.vehicles.streaming.sink.repository;

import com.vmware.tanzu.data.IoT.vehicles.domains.VehicleTelemetry;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface VehicleTelemetryRepository extends CrudRepository<VehicleTelemetry,String> {
}

File: components/iot-connected-vehicles-repository/src/main/java/com/vmware/tanzu/data/IoT/vehicles/telemetry/repositories/VehicleTelemetryRepository.kt
package com.vmware.tanzu.data.IoT.vehicles.telemetry.repositories

import com.vmware.tanzu.data.IoT.vehicles.domains.VehicleTelemetry
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface VehicleTelemetryRepository : CrudRepository<VehicleTelemetry,Long> {
}